llvm/unittests/IR/ConstantRangeListTest.cpp

```diff
@@ -94,4 +94,122 @@ TEST_F(ConstantRangeListTest, Insert) {
   EXPECT_TRUE(CRL == Expected);
 }
 
+ConstantRangeList GetCRL(const SmallVector<std::pair<APInt, APInt>, 2> &Pairs) {
```

Comment:
does ArrayRef work here?

Reply:
Ah, it works! I thought ArrayRef requires specifying the explicit type at callsites.
Then, with ArrayRef, each callsite creates a temp std::vector var by default?